installer/templates/phx_assets/webpack.config.js

```diff
@@ -45,9 +37,19 @@ module.exports = (env, options) => {
       ]
     },
     plugins: [
-      new MiniCssExtractPlugin({ filename: '../css/[name].css' }),
-      new CopyWebpackPlugin([{ from: 'static/', to: '../' }])
-    ]
-    .concat(devMode ? [new HardSourceWebpackPlugin()] : [])
+      new MiniCssExtractPlugin({ filename: '../css/app.css' }),
+      new CopyWebpackPlugin({
+        patterns: [
+          { from: 'static/', to: '../' }
+        ]
+      })
+    ],
+    optimization: {
+      minimizer: [
+        new TerserPlugin({}),
+        new CssMinimizerPlugin({})
+      ]
+    },
+    devtool: devMode ? 'eval-cheap-module-source-map' : undefined
```

Comment:
The [Terser plugin page](https://webpack.js.org/plugins/terser-webpack-plugin/#note-about-source-maps) mentions:

> **Note about source maps**
> Works only with `source-map`, `inline-source-map`, `hidden-source-map` and `nosources-source-map` values for the [devtool](https://webpack.js.org/configuration/devtool/) option.

Can't confirm but just a heads up.

Reply:
Good point, which of the options should we pick then, if `eval-cheap-module-source-map` is really not accepted anymore?

Reply:
Not my area of expertise, but could look into [SourceMapDevToolPlugin](https://webpack.js.org/plugins/source-map-dev-tool-plugin/) instead.

Reply:
Not my expertise either.

My suggestion would be:
```
const webpack = require('webpack');
...
plugins: [
  new webpack.EvalSourceMapDevToolPlugin({
     exclude: ['./vendor/**/*.js']
   })
]
devtool: false
```

This would create inline source maps...

Reply:
@Gazler
I've replaced `eval-cheap-module-source-map` with `source-map` for now again.
Because Terser plugin doesn't work with `eval-cheap-module-source-map`. Just to keep it simple first.

But would be great if we could improve the build time later again.

Reply:
@cw789 I guess we only need Terser in production, right? So couldn't we disable Terser in dev and use cheap source maps for dev?

Reply:
I think the point is it would at least need EvalSourceMapDevToolPlugin to have `eval-cheap-module-source-map` available again...